src/main/java/com/hellosign/sdk/resource/TemplateDraft.java

```diff
@@ -145,9 +150,19 @@ public boolean removeSignerRole(String signerRole) throws HelloSignException {
      *         field.
      */
     public void addMergeField(String name, FieldType type) throws HelloSignException {
-        if (!FieldType.checkbox.equals(type) && !FieldType.text.equals(type)) {
+        List<FieldType> mergeableFieldTypes = Arrays.asList(
+            FieldType.CHECKBOX,
+            FieldType.TEXT
+        );
+
+        if (!mergeableFieldTypes.contains(type)) {
+            List<String> asStrings = mergeableFieldTypes
+                .stream()
+                .map(Enum::toString)
+                .collect(Collectors.toList());
             throw new HelloSignException(
-                "Only 'text' or 'checkbox' types allowed for merge fields.");
+                String.format("Only %s types allowed for merge fields.", String.join(",", asStrings))
+            );
```

Comment:
This change was unnecessary since we're not actually adding any other types. I might revert this part.